src/status_im/utils/universal_links/core.cljs

```diff
@@ -46,6 +46,10 @@
   (boolean
    (re-matches #"((^https?://get.status.im/)|(^status-im://))[\x00-\x7F]+$" url)))
 
+(defn deep-link? [url]
+  (boolean
+   (re-matches #"(^status-im://)[\x00-\x7F]+$" url)))
```

Comment:
let's put those regex literals into constants.
also, can we have some unit tests for regexes?